<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientInterface</class>
 <widget class="QWidget" name="ClientInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ButtonAll">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>380</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOne">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TheTableWidgetUserList">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>300</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="TheTableWidgetMessagePage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>430</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>398</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditMessage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonSend">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>198</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="TheChatUser">
      <property name="text">
       <string>you are talking to :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TheLabel">
      <property name="text">
       <string>everyone</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
